src/elements/paginator/common/paginator-common.ts

```diff
@@ -114,6 +112,10 @@ export const SbbPaginatorCommonElementMixin = <T extends AbstractConstructor<Lit
       );
     }
 
+    protected currentPageLabel(): string {
```

Comment:
now it could be private?
```suggestion
    private _currentPageLabel(): string {
```